server/base/src/main/java/org/apache/accumulo/server/problems/ProblemReports.java

```diff
@@ -217,9 +218,9 @@ private Iterator<Entry<Key,Value>> getIter2() {
                 scanner.setTimeout(3, TimeUnit.SECONDS);
 
                 if (table == null) {
-                  scanner.setRange(new Range(new Text("~err_"), false, new Text("~err`"), false));
+                  scanner.setRange(ProblemSection.getRange());
                 } else {
-                  scanner.setRange(new Range(new Text("~err_" + table)));
+                  scanner.setRange(new Range(new Text(ProblemSection.getRowPrefix() + table)));
```

Comment:
This can avoid a Text:

```suggestion
                  scanner.setRange(new Range(ProblemSection.getRowPrefix() + table));
```

Reply:
I found two more instances of `new Text` use in ProblemReports and was able to remove all three and the import `Text` statement.